models/user.dao.js

```diff
@@ -0,0 +1,52 @@
+const { AppDataSource } = require("./dataSource");
+
+  const createuser = async (email, password, firstname, lastname, skintype) => {
+    
+    try{
+   
+    return AppDataSource.query(
+      `
+      INSERT INTO users (
+      email,
+      password, 
+      first_name,
+      last_name,
+      skin_type
+    ) VALUES (
+      ?,
+      ?,
+      ?,
+      ?,
+      ?
+    )
+    `, 
+    [email, password, firstname, lastname, skintype]
+    );
+    }  catch(err) {
+    const error= new Error('INVAID_DATA_INPUT');
+    error.statusCode = 500;
+    throw error;
```

Comment:
해당 코드 라인을 보면 들여쓰기가 하나도 적용되어 있지 않습니다.
어떤 부분이 하위 코드 블럭의 내용인지 한눈에 보기 어렵습니다.

어제 구두로 설명드린 내용을 적용할 수 있도록 꼼꼼하게 확인해주세요

```suggestion
    return AppDataSource.query(
      `
      INSERT INTO users (
        email,
        password, 
        first_name,
        last_name,
        skin_type
      ) VALUES (
        ?,
        ?,
        ?,
        ?,
        ?
      )
      `, 
        [email, password, firstname, lastname, skintype]
      );
    } catch(err) {
      const error= new Error('INVAID_DATA_INPUT');
      error.statusCode = 500;
      throw error;
```